nucypher-core-wasm/src/lib.rs

```diff
@@ -105,6 +105,55 @@ where
         .unchecked_into::<T>()
 }
 
+macro_rules! generate_from_bytes {
+    ($struct_name:ident) => {
+        #[wasm_bindgen]
+        impl $struct_name {
+            #[wasm_bindgen(js_name = "fromBytes")]
+            pub fn from_bytes(bytes: &[u8]) -> Result<$struct_name, Error> {
+                from_bytes(bytes).map(Self)
+            }
+        }
+    };
+}
```

Comment:
Looking good!  It's not too abstract and is nice encapsulation of boilerplate for bindings.